### ISSUE TITLE
Rename `change_feed` methods and implement proper stopping when changing type of feed

### ISSUE THREAD

**Issue by Wanchai290** [Wanchai290/tmita-optical-cardyomyocyte-analysis#8]
`LiveFeedWrapper` is just the product of unfinished code, it was created just to fix the fact you couldn't properly switch the live video feed between the frames provider server and the live video feed from a camera

**Comment by Wanchai290** [comment]
The issue has changed : having `LiveFeedWrapper` is actually the only way to have an actual object on which you can call a method such as `object.change_feed()`. If we put this kind of method in `AbstractFrameProvider`, it actually has no sense because an object implementing it (such as `VideoReader`) cannot change its own instance. Thus the existence of `LiveFeedWrapper`.

tl;dr we need `LiveFeedWrapper` to encapsulate objects of type `AbstractFrameProvider` so that the instance of the latter can be replaced

**Comment by Wanchai290** [comment]
Resolved by #12
